packages/gatsby-source-wordpress/src/steps/create-schema-customization/transform-fields/transform-union.js

```diff
@@ -42,9 +42,7 @@ export const transformListOfUnions = ({ field, fieldName }) => {
       }
 
       return resolvedField.reduce((accumulator, item) => {
-        // @todo use our list of Gatsby node types to do a more performant check
```

Comment:
Do you want to keep this todo?

Reply:
I don't think it's true anymore because the `item` var doesn't contain the full node so we need to grab it regardless